src/transformers/trainer.py

```diff
@@ -470,7 +470,10 @@ def train(self, model_path: Optional[str] = None):
                     steps_trained_in_current_epoch -= 1
                     continue
 
-                tr_loss += self._training_step(model, inputs, optimizer)
+                loss = self._training_step(model, inputs, optimizer)
+                tr_loss += loss
+
+                epoch_iterator.set_postfix(loss=loss)
```

Comment:
Will not be included in this commit, but I increasingly find myself applying this patch to this and other training pipelines so that I can easily tell whether my model checkpoint has loaded and trains properly.